### DISCUSSION TITLE
Install and store PocketBase on external drive

### DISCUSSION THREAD

**Discussion by NicoCaldo** [pocketbase/pocketbase#1336] [Q&A]
So I have a SBC with limited amount of memory for storage. If I normally install it as described on the [documentation](https://pocketbase.io/docs/) everything works perfect as it will be installed in the same partition as the OS (in my case Linux Ubuntu) is.

Now, I have an SD card connected to it and I can create partitions on it. I've seen that PocketBase [let you store data on Amazon S3](https://pocketbase.io/docs/files-handling/#storage-options).

Is there an option to tell PocketBase to store data on another partition than the one where PocketBase is installed?

**Comment by ganigeorgiev** [comment]
You can specify where your local `pb_data` would be with the `--dir` flag:

```
./pocketbase serve --dir='/path/to/sdcard/pb_data'
```
_(you can explore all available flags with `--help`)_
